cookbooks/cdo-ruby/test/integration/default/inspec/ruby_spec.rb

```diff
@@ -0,0 +1,6 @@
+require_relative '../../../shared/helper_spec'
+
+file_exist '/usr/local/bin/ruby'
+cmd 'ruby -v', '3.3.4'
+cmd 'gem -v', '3.5.17'
+cmd 'bundler -v', '2.5.17'
```

Comment:
This is a conversion of the (not turned on) serverspec. Its now enabled and will make sure Chef verifies its upgrades worked.